### DISCUSSION TITLE
Welcome to delta Discussions!

### DISCUSSION THREAD

**Discussion by dennyglee** [delta-io/delta#3769] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

* Ask questions you’re wondering about. Any serious question is fair game (though let's try to stay on topic for `delta` otherwise I'll talk about cycling or coffee too much)
* Share ideas. Share thoughts. 
* Engage with other community members.
* Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by felipepessoto** [comment]
Hey Denny Lee, it is great to see this new place to discuss. I've never used the Discussions feature before, but it seems nice to have everything integrated in the GitHub, sometimes I forget we have a Slack channel. Is that the plan to move the Slack community to here? Sometimes I see interesting discussions in Slack that folks using GitHub only would miss.

My name is Felipe, I'm a software engineer and I have been working with Delta for the past 2 years. I'm an admirer of OSS, I used to work with .NET and sent a few small contributions in the past, and now Delta is my favorite OSS project :) which I have sent some PRs, and I try to participate in design discussions always I have time.

Denny, would be great to also hear about you.

Thanks.

**Reply by dennyglee** [reply]
Hey @felipepessoto - based on our experiences using discussions in `delta-rs` and `delta-incubator`, I'm inclined to agree that we can *move* more of our discussions here.  It's often easier to find previous threads in GitHub discussions and we can fork various threads to help the decision making.  As well, once we do that, its easier for our issues and PRs to reference the discussions and vice versa.    And welcome aboard! :)

**Comment by newfront** [comment]
👋 My name is Scott Haines. I've been using Delta for the past 6 years and more recently just finished up co-authoring the [Delta Lake Definitive Guide](https://www.oreilly.com/library/view/delta-lake-the/9781098151935/) with Denny, Tristian, R Tyler, and Prashanth. I enjoy teaching people how to engineer high-performance, high-trust streaming data applications and love Java, Scala, and became a Python convert via pyspark more recently. I'm new to Rust but enjoy learning new languages. You can find me on Linkedin (https://www.linkedin.com/in/scotthaines/) if you would like to connect outside of github or slack.

**Reply by felipepessoto** [reply]
Hey Scott. Is the version of the book at O’Reilly already the final one? Just confirming because the cover says early access

**Reply by newfront** [reply]
The early release at OReilly is ER5, we made significant changes and the final book will be out on OReilly and Kindle shop early November, and print is closer to the end of November!
